tests/Behat/Context/FixtureContext.php

```diff
@@ -44,6 +46,31 @@ public function theHasAContentElementWithContent($type, $pageTitle, $elementTitl
         }
     }
 
+    /**
+     * @Given /a "([^"]+)" "([^"]+)" with a "([^"]+)" searchable fields block/
+     *
+     * @param string $pageTitle
+     * @param string $type
+     * @param string $elementTitle
+     */
+    public function thePageHasASearchableFieldsBlock($type, $pageTitle, $elementTitle)
+    {
+        $elementalArea = $this->getElementalArea($type, $pageTitle);
+        $elementalArea->Elements()->add(
+            $this->getFixtureFactory()->createObject(ElementalSearchableFieldsBlock::class, $elementTitle)
+        );
+    }
+
+    /**
+     * @Given /a "([^"]+)" searchable fields object/
+     *
+     * @param string $title
+     */
+    public function thePageHasASearchableFieldsObject($title)
+    {
+        $this->getFixtureFactory()->createObject(ElementalSearchableFieldsObject::class, $title);
+    }
```

Comment:
You should be able to use `Given a "SilverStripe\FrameworkTest\Elemental\Model\ElementalSearchableFieldsObject" "My title here"` without inventing a new method.
There's a whole bunch of general-purpose methods in [behat-extension's `FixtureContext`](https://github.com/silverstripe/silverstripe-behat-extension/blob/5/src/Context/FixtureContext.php) for exactly this purpose.

Reply:
This is especially pertinent given https://github.com/silverstripe/silverstripe-frameworktest/pull/188#discussion_r1645250484